First edit for partner registration (#1972)
MicrosoftDocs/microsoft-365-docs: microsoft-365/managed-desktop/get-started/register-devices-partner.md

If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or the OEM name, serial number, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 

For more information about Autopilot in Partner scenarios, see [Add devices to a customer’s account] (https://docs.microsoft.com/partner-center/autopilot#add-devices-to-a-customers-account).

## Register devices by using the OEM API

Before completing registration for a customer, you must first establish a relationship with them. You should have a unique link to provide to your respective customers. See [How to establish OEM relationship] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#oem-authorization).

Once you've established the relationship, you can start registering devices for customers using the Group Tag **Microsoft365Managed_Autopilot**.

> [!IMPORTANT]

> > The group name must **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.

Reviewer: ```suggestion
> The group name must match **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```

## Prepare for registration 

> Before completing registration for a customer, you must first establish a relationship with them at the [Partner Center] (https://partner.microsoft.com/dashboard). See the [consent documentation] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#csp-authorization) for more details on that process. Any CSP partner can add devices on behalf of any customer, as long as the customer consents. You can also learn more about partner relationship and Autopilot permissions at [Partner Center help] (https://docs.microsoft.com/partner-center/customers_revoke_admin_privileges#windows-autopilot)

Reviewer: ```suggestion
Before completing registration for a customer, you must first establish a relationship with them at the [Partner Center] (https://partner.microsoft.com/dashboard). See the [consent documentation] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#csp-authorization) for more details on that process. Any CSP partner can add devices on behalf of any customer, as long as the customer consents. You can also learn more about partner relationships and Autopilot permissions at [Partner Center help] (https://docs.microsoft.com/partner-center/customers_revoke_admin_privileges#windows-autopilot).
```

Reviewer: Please change "relationship" to "relationships" in the last sentence and add a final period (.) as indicated.

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> This documentation is only for Partners and OEMs. The process for self-registration is documented in [Register devices in Microsoft Managed Desktop yourself](register-devices-self.md).

## Register devices by using Partner Center

> Once you have established the relationship with your customers, you can leverage Partner Center to add devices to Autopilot for any of the customers that you have a relationship with following the below steps :

Reviewer: ```suggestion
Once you have established the relationship with your customers, you can leverage Partner Center to add devices to Autopilot for any of the customers that you have a relationship with by following these steps:
```

1. Navigate to [Partner Center](https://partner.microsoft.com/dashboard)
2. Select **Customers** from the Partner Center menu and then select the customer whose devices you want to manage.
3. On the customer's detail page, select **Devices**.

> 4. Under **Apply profiles** to devices select **Add devices**.

Reviewer: ```suggestion
4. Under **Apply profiles** to devices, select **Add devices**.
```

5. Enter **Microsoft365Managed_Autopilot** for the Group Name and then select **Browse** to upload the customer's list (in .csv file format) to Partner Center.

> >[!IMPORTANT]

Reviewer: ```suggestion
> [!IMPORTANT]
```

> >The Group Name must match exactly **Microsoft365Managed_Autopilot** including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.

Reviewer: ```suggestion
> The Group Name must match **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```

>[!NOTE]

> > You should have received this .csv file with your device purchase. If you didn't receive a .csv file, you can create one yourself by following the steps in [Adding devices to Windows Autopilot] (https://docs.microsoft.com/windows/deployment/windows-autopilot/add-devices#collecting-the-hardware-id-from-existing-devices-using-powershell). The Windows PowerShell Ssript is different from the one used for the [Microsoft Managed Desktop Admin portal] (https://docs.microsoft.com/microsoft-365/managed-desktop/get-started/register-devices-self?view=o365-worldwide#obtain-the-hardware-hash). Partners should use [Get-WindowsAutoPilotInfo] (https://www.powershellgallery.com/packages/Get-WindowsAutoPilotInfo) to register devices for Microsoft Managed Desktop devices in Partner Center.

Reviewer: ```suggestion
> You should have received this .csv file with your device purchase. If you didn't receive a .csv file, you can create one yourself by following the steps in [Adding devices to Windows Autopilot] (https://docs.microsoft.com/windows/deployment/windows-autopilot/add-devices#collecting-the-hardware-id-from-existing-devices-using-powershell). The Windows PowerShell script is different from the one used for the [Microsoft Managed Desktop Admin portal] (https://docs.microsoft.com/microsoft-365/managed-desktop/get-started/register-devices-self?view=o365-worldwide#obtain-the-hardware-hash). Partners should use [Get-WindowsAutoPilotInfo] (https://www.powershellgallery.com/packages/Get-WindowsAutoPilotInfo) to register devices for Microsoft Managed Desktop devices in Partner Center.
```

Reviewer: Please fix the typo in "PowerShell Sscript" as indicated.

If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or the OEM name, serial number, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 

For more information about Autopilot in Partner scenarios, see [Add devices to a customer’s account] (https://docs.microsoft.com/partner-center/autopilot#add-devices-to-a-customers-account).

## Register devices by using the OEM API

Before completing registration for a customer, you must first establish a relationship with them. You should have a unique link to provide to your respective customers. See [How to establish OEM relationship] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#oem-authorization).

Once you've established the relationship, you can start registering devices for customers using the Group Tag **Microsoft365Managed_Autopilot**.

> >[!IMPORTANT]

Reviewer: ```suggestion
> [!IMPORTANT]
```

> >The group name must match exactly **Microsoft365Managed_Autopilot** including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.

Reviewer: ```suggestion
> The group name must **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```
